client/src/app/management/components/organisation-tag-dialog/organisation-tag-dialog.component.html

```diff
@@ -0,0 +1,27 @@
+<h2 mat-dialog-title>{{ (isCreateView ? 'Create an organization tag' : 'Edit an organization tag') | translate }}</h2>
+<div mat-dialog-content class="organization-tag-content">
+    <form [formGroup]="organisationTagForm">
+        <mat-form-field appearance="fill">
+            <mat-label>{{ 'Name' | translate }}</mat-label>
+            <input #name matInput formControlName="name" required />
+        </mat-form-field>
+        <mat-form-field appearance="fill">
+            <mat-label>{{ 'Color' | translate }}</mat-label>
+            <input matInput formControlName="color" [required]="isCreateView" />
```

Comment:
Please create a custom validation. Maybe ther is an existing regexvalidator you can use. The regex would be `^[0-9a-f]{6}$` if i'm not mistaken.

Reply:
Currently I can enter too many or too less characters and also invalid ones.

Reply:
> Please create a custom validation. Maybe ther is an existing regexvalidator you can use. The regex would be `^[0-9a-f]{6}$` if i'm not mistaken.

Yes, that's right - I will extend this to `^[0-9a-fA-F]{6}$`.